src/App.tsx

```diff
@@ -9,13 +9,18 @@ function App() {
     // TODO
   };
 
+  const handleRemoveTodo = (id: string) => {
+    console.log(id)
```

Comment:
```suggestion
```
Great job @Jeffersonlpu, well done!
Could You please remove this console.log?

Reply:
Thanks, @jhrodriguesit ! The console.log was already removed!